micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSAddV2.js

```
@@ -86,27 +84,103 @@
       const jsonPath = api?.responseJson ? api?.responseJson : "mdms[0].id";
       setShowToast(`${t("WBH_SUCCESS_MDMS_MSG")} ${_.get(resp, jsonPath, "NA")}`);
-      closeToast();
-      gotoView();
+      setShowErrorToast(false);
 
-      //here redirect to search screen(check if it's required cos user might want  add multiple masters in one go)
+      // Build messages array from additionalProperties for localization
+      const messages = [];
+      if (additionalProperties && typeof additionalProperties === "object") {
+        for (const fieldName in additionalProperties) {
+          if (additionalProperties.hasOwnProperty(fieldName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)